src/Simulation/Simulators/Microsoft.Quantum.Simulators.csproj

```diff
@@ -23,6 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Quantum.Telemetry" Version="0.0.9-alpha" />
```

Comment:
If we keep the version number of the telemetry library in sync with the QDK version, it makes most sense to set the same version number here as we set for all other packages produced on the compiler repo. This makes it easier to do a search and replace when we need to update the packages for the local repo build.

Reply:
@vxfield, FYI.